boomerang.js

```diff
@@ -1520,6 +1528,9 @@ BOOMR_check_doc_domain();
 				xhr = new (BOOMR.orig_XMLHttpRequest || BOOMR.window.XMLHttpRequest)();
 				xhr.open("POST", impl.beacon_url);
 				xhr.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
+				if( impl.beacon_auth_token !== "") {
```

Comment:
The beacon_auth_key defaults to 'Authorization' (line 248).  Would you still like an empty check?  No Problem if you do ;-).